core/templates/dev/head/services/SearchServiceSpec.ts

```diff
@@ -16,29 +16,72 @@
  * @fileoverview Tests that average ratings are being computed correctly.
  */
 
+import { SearchService } from './SearchService';
+import { EventService } from './EventService';
+import { LoggerService} from './LoggerService';
+import { ErrorHandler } from '@angular/core';
+import { HttpClient, HttpEvent, HttpHandler,
+  HttpRequest } from '@angular/common/http';
+import { Observable } from 'rxjs';
+import {
+  MissingTranslationHandler, MissingTranslationHandlerParams,
+  TranslateCompiler,
+  TranslateLoader,
+  TranslateParser,
+  TranslateService,
+  TranslateStore
+} from '@ngx-translate/core';
+
 // TODO(#7222): Remove the following block of unnnecessary imports once
 // the code corresponding to the spec is upgraded to Angular 8.
 import { UpgradedServices } from 'services/UpgradedServices';
 // ^^^ This block is to be removed.
 
-require('services/SearchService.ts');
-
 describe('Search service', function() {
-  var SearchService;
+  var searchService: SearchService;
 
-  beforeEach(angular.mock.module('oppia'));
   beforeEach(angular.mock.module('oppia', function($provide) {
     var ugs = new UpgradedServices();
     for (let [key, value] of Object.entries(ugs.upgradedServices)) {
       $provide.value(key, value);
     }
   }));
-  beforeEach(angular.mock.inject(function($injector) {
-    SearchService = $injector.get('SearchService');
-  }));
+
+  beforeEach(() => {
+    searchService = new SearchService(new EventService(),
+      new LoggerService(new ErrorHandler()),
+      new HttpClient(new class extends HttpHandler {
+        handle(req: HttpRequest<any>): Observable<HttpEvent<any>> {
```

Comment:
Can you try to drop the `<any>` part from `HttpRequest` and `HttpEvent`?

Reply:
I can't drop any from here since this is coming straight from angular and it's not something I created so it only be any or null when I inspected the package so we should go with any...
![Screenshot 2019-10-17 at 2 40 37 PM](https://user-images.githubusercontent.com/30312043/66995136-1eba8880-f0ec-11e9-8dbf-4c5037801d5b.png)

Reply:
Okay then, lets leave it as-is.